site.yml

```diff
@@ -3,10 +3,32 @@
 
 - hosts: "{{ hosts_prefix }}-tsuru-db"
   sudo: yes
+  vars:
+    mongodb_conf_dbpath: "/var/lib/mongodb"
   roles:
     - bennojoy.redis
     - greendayonfire.mongodb
 
+# MongoDB: migrate old /data/db dir to new location
+  post_tasks:
+    - name: MongoDB | Check /data/db directory
+      stat: path=/data/db
+      register: mongo_data_db
+    - name: MongoDB | Migrate data - stop mongo
+      service: name=mongod state=stopped
+      when: mongo_data_db.stat.isdir is defined and mongo_data_db.stat.isdir
```

Comment:
What happens if you omit `mongo_data_db.stat.isdir is defined and` here? What does that part of the condition guard against?